cblox/include/cblox/io/tsdf_submap_io_inl.h

```diff
@@ -8,65 +8,83 @@
 #include <glog/logging.h>
 
 #include "./QuatTransformation.pb.h"
-#include "./TsdfSubmap.pb.h"
-#include "./TsdfSubmapCollection.pb.h"
-
-#include "cblox/utils/quat_transformation_protobuf_utils.h"
+#include "./Submap.pb.h"
+#include "./SubmapCollection.pb.h"
 
 namespace cblox {
 namespace io {
 
-template <typename SubmapType>
+// we assume most SubmapTypes to have an ESDF map
```

Comment:
I guess since now both layers are always in the proto message, it makes sense that we just try to read everything (it shouldn't matter if it has only tsdf or only esdf or nothing, the number of blocks in the proto header will just be 0 then)

Reply:
Ok now I see what you meant :)